ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/transaction/GoQuorumPrivateTransaction.java

```diff
@@ -49,6 +52,11 @@ public static GoQuorumPrivateTransaction from(
       throw new IllegalArgumentException("Transaction does not contain a valid privateFor list.");
     }
 
+    if (transactionJsonParameters.value().isPresent()
+        && transactionJsonParameters.value().get().compareTo(BigInteger.ZERO) > 0) {
```

Comment:
could that value be negativ? We could just check whether the value is ZERO ...

Reply:
can you transfer negative amounts?

Reply:
Makes sense to reject it either way.